static/app/views/releases/detail/overview/releaseAdoption.tsx

```diff
@@ -198,67 +206,96 @@ function ReleaseComparisonChart({
     allowEmptyPeriod: true,
   });
 
+  const adoptionStage = release.adoptionStages?.[project.slug].stage;
+  const adoptionStageLabel = ADOPTION_STAGE_LABELS[adoptionStage];
+  const allEnvironments = environment.length === 0;
+  const multipleEnvironments = environment.length > 1;
+
   return (
-    <RelativeBox>
-      <ChartLabel top="0px">
-        <ChartTitle>
-          {t('Sessions Adopted')}
-          <QuestionTooltip
-            position="top"
-            title={t(
-              'Adoption compares the sessions of a release with the total sessions for this project.'
-            )}
-            size="sm"
+    <Wrapper>
+      <Feature features={['release-adoption-stage']}>
+        <SectionHeading>
+          {allEnvironments
+            ? t('All Environments')
+            : multipleEnvironments
+            ? t(`${environment.join(', ')}`)
+            : t(`${environment}`)}
+        </SectionHeading>
+        <SidebarSectionTitle title={t('Adoption Stage')}>
```

Comment:
Should the component work like this
`<SidebarSectionTitle>{t('Adoption Stage')}</SidebarSectionTitle>`
instead of wrapping all children?